initial-ai-src/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ruter Reis Discount Calculator</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Ruter Reis Discount Calculator</h1>
+      <p>Find out what discount you can get on Oslo public transport tickets</p>
+    </header>
+
+    <main>
+      <div class="calculator-container">
+        <div class="input-section">
+          <label for="trips-per-week"
+            >How many trips do you expect per week?</label
```

Comment:
❓ Why default to "trips per week" as the question? 

I can imagine that it's easier for people to think about per-week, but also Reis is rolling over 30 days. It might make more sense to ask for 30 days, and average it out.